src/components/atoms/TextArea/TextArea.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import cn from 'classnames';
+
+import './textArea.css';
+
+const TextArea = (props: Props) => {
+  const {
+    disableCopyPaste,
+    showError,
+    label,
+    error,
+    labelStyle,
+    errorStyle,
+    textAreaStyle,
+    parentDivClass,
+    ...restProps
+  } = props;
+
+  const textAreaClassName = cn({
+    "txta37Area": true,
+    "txta37Normal": !showError,
+    "txta37Error": showError
+  });
+
+  return (
+    <div className={`txta37Div ${parentDivClass}`}>
+
+      {
+        label &&
+        <div style={labelStyle} className="txta37Label">
+          {label}
+        </div>
+      }
+
+      <textarea
+        {...restProps}
+        className={textAreaClassName}
+        style={textAreaStyle}
+        onCopy={disableCopyPaste ? (e) => onCopy(e) : () => { }}
+        onPaste={disableCopyPaste ? (e) => onPaste(e) : () => { }}
+      />
+
+      {
+        showError &&
```

Comment:
showError and error can be only one prop. Only error prop can determine if there is an error. If error === "", then don't show error.